tools/packaging/kernel/build-kernel.sh

```diff
@@ -17,7 +17,7 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 kata_version="${kata_version:-}"
 
 #project_name
-readonly project_name="kata-containers"
+readonly project_name="kata-containers/kata-containers/tools"
```

Comment:
please don't modify this variable or the install path will change

https://github.com/kata-containers/kata-containers/blob/85584427e08d3615e35330dc51f9db1cf2b8079c/tools/packaging/kernel/build-kernel.sh#L405

Reply:
ok